react-ui/src/styleTransfer.js

```diff
@@ -0,0 +1,60 @@
+import * as tf from '@tensorflow/tfjs'
+tf.ENV.set('WEBGL_PACK', false)
+
+let styleModel, transferModel
+
+const styleModelPath = 'tf_models/style/model.json'
+const transferModelPath = 'tf_models/transfer/model.json'
+
+export default async function transformImageWithStyle(imageData, style) {
+
+  transferModel = transferModel || await tf.loadGraphModel(transferModelPath);
+
+  const stylized = await tf.tidy(() => {
+    const styleInput = styleFromArray(style)
+    const imgInput = tf.browser.fromPixels(imageData).toFloat().expandDims()
+    return transferModel.predict([imgInput, styleInput]).squeeze().div(tf.scalar(255.0))
+  })
+  return new ImageData(await tf.browser.toPixels(stylized),imageData.width,imageData.height) // returns Uint8ClampedArray
+}
+
+export async function getImageStyle(imageData) {
+
+  styleModel = styleModel || await tf.loadGraphModel(styleModelPath)
+
+  return await tf.tidy(() => {
+    const imgInput = tf.browser.fromPixels(imageData).toFloat().expandDims()
+    var res = styleToArray(styleModel.predict(imgInput).expandDims())
```

Comment:
`var` is not really used in new-style code. In this case, using `const` would work just as well